kotest-property/src/commonMain/kotlin/io/kotest/property/arbitrary/longs.kt

```diff
@@ -34,13 +47,20 @@ fun Arb.Companion.positiveLong(max: Long = Long.MAX_VALUE): Arb<Long> = long(1L,
  */
 fun Arb.Companion.negativeLong(min: Long = Long.MIN_VALUE): Arb<Long> = long(min, -1L)
 
-class LongShrinker(private val range: LongRange) : Shrinker<Long> {
-   override fun shrink(value: Long): List<Long> = when (value) {
-      0L -> emptyList()
-      1L, -1L -> listOf(0L).filter { it in range }
+/**
+ * Returns an [Arb] that produces [LongArray]s where [generateArrayLength] produces the length of the arrays and
+ * [generateContents] produces the content of the arrays.
+ */
+fun Arb.Companion.longArray(generateArrayLength: Gen<Int>, generateContents: Arb<Long>): Arb<LongArray> =
```

Comment:
We tend to use the name somethingArb or somethingGen when composing arbs/gens.
So perhaps lengthGen and contentArb or something along those lines ?

Reply:
I am not sure how useful it is to repeat the type in the parameter name? I think `length`/`content` would be clear enough?

Reply:
Ok agreed.